_posts/2014-04-01-fun-with-webhooks.md

```diff
@@ -10,7 +10,7 @@ excerpt: Dan is working with us here at OnePageCRM. He's planning to make a 'dea
 ---
 
 
-This is Dan: <img class="img-responsive" src="/assets/images/authors/dan.jpg"/>
+This is Dan: <img alt ="OnePageCRM"  class="img-responsive" src="/assets/images/authors/dan.jpg"/>
```

Comment:
Can we spend some time and put descriptive alt tags? It is annoying but potentially a big SEO benifit. 

It will be easier to care about descriptive alt tags going forward if we spend a little bit of time to care about it now. 

No need to over think the alt tags. Just 1-4 words describing the image. If you can't think of anything, it is ok to just put "OnePageCRM"